benches/run-generator.rs

```diff
@@ -46,7 +57,7 @@ fn run(c: &mut Criterion) {
                         11000000000,
                         ALLOW_BACKREFS,
                     );
-                    assert!(conds.is_ok());
+                    let _ = black_box(conds);
```

Comment:
Doesn't this ignore the error if there is one?

Reply:
yes. some of the generators I included fail. i.e. they exceed the cost limit. I think it's reasonable to have benchmarks that max out the cost. In principle, there could be a valid generator that has a cost just short of this